packages/shopify-codemod/transforms/global-assignment-to-default-export.js

```diff
@@ -0,0 +1,47 @@
+export default function globalAssignmentToDefaultExport({source}, {jscodeshift: j}, {printOptions = {}, appGlobalIdentifiers}) {
+  function findFirstMember(memberExpression) {
+    if (j.MemberExpression.check(memberExpression)) {
+      return findFirstMember(memberExpression.object);
```

Comment:
Yeah, I guess we'd just put `Sello` as the `appGlobalIdentifiers`, I was thinking we'd do something like `Sello.Components` but that's unnecessary. KCSO :+1: